packages/theme/src/types.ts

```diff
@@ -43,7 +43,10 @@ type PaletteAdditions = {
   linkHover: string;
   link: string;
   gold: string;
-  sidebar: string;
+  sidebar: {
```

Comment:
If you're up for extending this change a bit I think we should rename this, since sidebar is a bit of an implementation detail. Something like `topNav` or `navigation` would make sense I think, what do others think?

Reply:
Yep, fair enough. I think I'd go with `navigation` just so consistent with Material-UI (have pushed changes to that effect) but more than happy to take direction here if there is a preference.